### ISSUE TITLE
Friendly error message for Optifine and potentially other incredibly popular mods

### ISSUE THREAD

**Issue by ekevoo** [toolbox4minecraft/amidst#205]
The amount of bug reports including Optifine is impressive.

![Too damn high!](https://imgflip.com/s/meme/Too-Damn-High.jpg)

Maybe before crashing and burning with a huge stack trace that only the initiated can understand, we should look at things like profile name and library names for, say, common bad words? :)

**Comment by stefandollase** [comment]
It is impressive, indeed! This is definitely going into the 4.2 release :-P

**Comment by ArgTang** [comment]
@stefandollase Could i get feedback on the pull rewuest for this issue?

**Comment by stefandollase** [comment]
@ArgTang
I reviewed your pull request. The idea should work, but there are still some rough edges that need to be worked out.

A more elaborate fix would depend on the changes that I proposed [here](https://github.com/toolbox4minecraft/amidst/issues/146#issuecomment-208092539). However, I did not yet get the time to implement these changes. They would allow us to use the vanilla Minecraft version that the modded Minecraft version is based on.
